web/package.json

```diff
@@ -8,7 +8,13 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "test:unit": "jest",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint ./src --fix",
+    "prettier": "prettier src --check",
+    "prettier:fix": "prettier src --write",
+    "format": "yarn prettier && yarn lint",
+    "format:fix": "yarn prettier:fix && yarn lint:fix",
+    "test": "yarn concurrently -g 'yarn:test:*'",
```

Comment:
I couldn't figure out how to get prettier to `--fix` as part of eslint so we're just going to run it separately. The workflow will be:

1. Run prettier 
2. Run eslint 

I added a plugin to eslint to avoid conflicting rules with prettier so this should work. 

The commands you'll use locally will probably be `yarn format` to test and `yarn format:fix` to fix. Open to using a better work that means "prettier + lint"